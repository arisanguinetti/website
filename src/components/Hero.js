import React, { Component } from "react";

class Hero extends Component {
  render() {
    return (
      <header id="home" className="hero parallax-yes">
        <div id="parallax-1" className="parallax">
          <div
            className="bcg skrollable skrollable-between"
            data-anchor-target="#parallax-1"
            data-center="background-position: 50% 0px;"
            data-top-bottom="background-position: 50% -100px;"
            data-bottom-top="background-position: 50% 100px;"
          >
            <div className="hero-body">
              <div className="hero-text text-center">
                <h1>
                  I'm Ari Sanguinetti <span className="blinker" />
                </h1>

                <h1 className="small">Software Ninjaneer</h1>

                <p>32 years young, techy and nerd, music addict.</p>

                <div className="page-scroll">
                  <a href="#profile" className="btn btn-lg btn-custom">
                    Know Me Better
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </header>
    );
  }
}

export default Hero;
