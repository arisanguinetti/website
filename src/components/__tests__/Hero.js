import React from "react";
import { shallow } from "enzyme";
import Hero from "../Hero";

describe("Hero", () => {
  describe("Rendering", () => {
    it("should match to snapshot", () => {
      const component = shallow(<Hero />);
      expect(component).toMatchSnapshot();
    });
  });
});
