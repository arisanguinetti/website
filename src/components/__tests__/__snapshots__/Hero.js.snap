// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Rendering should match to snapshot 1`] = `
<header
  className="hero parallax-yes"
  id="home"
>
  <div
    className="parallax"
    id="parallax-1"
  >
    <div
      className="bcg skrollable skrollable-between"
      data-anchor-target="#parallax-1"
      data-bottom-top="background-position: 50% 100px;"
      data-center="background-position: 50% 0px;"
      data-top-bottom="background-position: 50% -100px;"
    >
      <div
        className="hero-body"
      >
        <div
          className="hero-text text-center"
        >
          <h1>
            I'm Ari Sanguinetti 
            <span
              className="blinker"
            />
          </h1>
          <h1
            className="small"
          >
            Software Ninjaneer
          </h1>
          <p>
             years young, techy and nerd, music addict.
          </p>
          <div
            className="page-scroll"
          >
            <a
              className="btn btn-lg btn-custom"
              href="#profile"
            >
              Know Me Better
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</header>
`;
